ROMFS/px4fmu_common/init.d/rc.sensors

```diff
@@ -93,6 +93,20 @@ then
 	sf1xx start -a
 fi
 
+# Lanbao PSK-CM8JL65-CC5 distance sensor
```

Comment:
Please use the generic UART configuration instead - see [this commit](https://github.com/PX4/Firmware/commit/b5e552924a998291820dd3741ddff41899dacbc5) for example (in particular the module.yaml).

Reply:
Upgraded driver startup with generic UART conf ([see here](https://github.com/Auterion/Firmware/commit/3650e6d88e4239aab3438ba3b1391024b76894ee)).
Tested on TELEM1 and TELEM2.